build/charts/theia/values.yaml

```diff
@@ -109,6 +109,11 @@ clickhouse:
         repository: "projects.registry.vmware.com/antrea/theia-zookeeper"
         pullPolicy: "IfNotPresent"
         tag: "3.8.0"
+      # -- Set securityContext
+      # Use a specific uid, gid for zookeeper
```

Comment:
Could you add `.` after each sentence? And also re-generate the doc.

Reply:
Fixed.